shopping-list-web/src/utilities/httpMethods/productMethods.ts

```diff
@@ -20,3 +20,10 @@ export const getCategoryDataResponseForProduct = async (product: string) => {
   const result = await response.json();
   return result;
 };
+
+export const getDataResponseForFilteredProducts = async (query: string): Promise<any> => {
```

Comment:
```suggestion
export const getDataResponseForFilteredProducts = async (query: string): Promise<string> => {
```

Reply:
Fixed the types here I think... but not sure how happy you'll be with the interface I made...

Reply:
I'm not that happy with it but I will allow it for now  😂